ui-admin/src/study/families/FamilyView.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import {
+  ParticipantTask,
+  StudyEnvironmentSurvey,
+  SurveyResponse
+} from 'api/api'
+import { StudyEnvContextT } from 'study/StudyEnvironmentRouter'
+import {
+  Link,
+  NavLink,
+  Route,
+  Routes
+} from 'react-router-dom'
+import ErrorBoundary from 'util/ErrorBoundary'
+import { NavBreadcrumb } from 'navbar/AdminNavbar'
+import LoadingSpinner from 'util/LoadingSpinner'
+
+import { Family } from '@juniper/ui-core'
+import {
+  navDivStyle,
+  navListItemStyle
+} from 'util/subNavStyles'
+import useRoutedFamily from './useRoutedFamily'
+import { FamilyOverview } from './FamilyOverview'
+import { uniq } from 'lodash/fp'
+import { FamilyMembersAndRelations } from 'study/families/FamilyMembersAndRelations'
+
+
+export type SurveyWithResponsesT = {
+  survey: StudyEnvironmentSurvey
+  response?: SurveyResponse
+  task: ParticipantTask
+}
+export type ResponseMapT = { [stableId: string]: SurveyWithResponsesT }
+
+/** loads an enrollee and renders the view for it */
+export default function FamilyView({ studyEnvContext }: { studyEnvContext: StudyEnvContextT }) {
+  const { isLoading, family, reload } = useRoutedFamily(studyEnvContext)
+  return <>
+    {isLoading && <LoadingSpinner/>}
+    {!isLoading && family && <LoadedFamilyView family={family} studyEnvContext={studyEnvContext}
+      onUpdate={reload}/>}
+  </>
+}
+
+/** shows a master-detail view for an enrollee with sub views on surveys, tasks, etc... */
+export function LoadedFamilyView({ family, studyEnvContext }:
+                                     { family: Family, studyEnvContext: StudyEnvContextT, onUpdate: () => void }) {
+  const { currentEnvPath } = studyEnvContext
+
+
+  return <div className="ParticipantView mt-3 ps-4">
+    <NavBreadcrumb value={family?.shortcode || ''}>
+      <Link to={`${currentEnvPath}/family/${family.shortcode}`}>
+        {family?.shortcode}</Link>
+    </NavBreadcrumb>
+    <div className="row">
+      <div className="col-12">
+        <h4>
+          {getFamilyNameString(family)} Family&nbsp;
+          <span className="detail" title="Participant shortcode"> ({family.shortcode})</span>
+        </h4>
+      </div>
+    </div>
+    <div className="row mt-2">
+      <div className="col-12">
+        <div className="d-flex">
+          <div style={navDivStyle}>
+            <ul className="list-unstyled">
+              <li style={navListItemStyle} className="ps-3">
+                <NavLink to="." className={getLinkCssClasses}>Overview</NavLink>
```

Comment:
```suggestion
                <NavLink end to="." className={getLinkCssClasses}>Overview</NavLink>
```
style nit: this will make it so the Overview item isn't always bolded, even when another section is selected

Reply:
😮  That has been bothering me so much!!!!